durablefunctionsmonitor-vscodeext/src/BackendProcess.ts

```diff
@@ -362,8 +367,6 @@ export class BackendProcess {
             }
         }
 
-        // Defaulting to 'func' command, hopefully it will be properly resolved
-        BackendProcess._funcExePath = 'func';
-        return BackendProcess._funcExePath;
+        return "";
```

Comment:
I'm afraid, there still can be scenarios when resolving Functions Host via PATH is the only valid option. 
E.g. some users might decide to install Functions Core Tools to a custom folder.
I suggest we still keep 'func' as a last chance option, and not fully rely on the above resolution algorithm.

This is exactly why I was suggesting to introduce some _probing call_ as a way to check for Core Tools.
Before doing the actual backend start command we could execute a quick probe like $"{funcExePath} --version".
No need to use child_process for that, it should be enough to just use execSync (see examples of using this method in other parts of the codebase). 
If that probing command succeeeds, then we're fine. If it fails, we show an error and don't go any further.
And it should be enough to run this probing command just once (and then cache the outcome).

Reply:
That makes sense. I misunderstood the default value of `funcExePath`. I updated the code to use `execSync`. Let me know if you have any other suggestions.